libs/application/templates/aosh/work-accident-notification/src/forms/Prerequisites.ts

```diff
@@ -0,0 +1,42 @@
+import { buildForm, buildSection } from '@island.is/application/core'
+import { Form, FormModes } from '@island.is/application/types'
+import { prerequisitesSection } from './WorkAccidentNotificationForm/prerequisitesSection'
+import { Logo } from '../assets/Logo'
+import { sections } from '../lib/messages'
+
+export const Prerequisites: Form = buildForm({
+  id: 'PrerequisitesForm',
+  title: '',
+  logo: Logo,
+  mode: FormModes.NOT_STARTED,
+  renderLastScreenButton: true,
+  renderLastScreenBackButton: true,
+  children: [
+    prerequisitesSection,
+    buildSection({
+      id: 'information',
+      title: sections.prerequisites.information,
+      children: [],
+    }),
+    buildSection({
+      id: 'accident',
+      title: sections.prerequisites.accident,
+      children: [],
+    }),
+    buildSection({
+      id: 'employee',
+      title: sections.prerequisites.employee,
+      children: [],
+    }),
+    buildSection({
+      id: 'overview',
+      title: sections.prerequisites.overview,
+      children: [],
+    }),
+    buildSection({
+      id: 'conclusion',
+      title: sections.prerequisites.conclusion,
+      children: [],
+    }),
```

Comment:
This is the way the reference template is setup along with multiple other application that are live and approved. https://github.com/island-is/island.is/blob/main/libs/application/templates/reference-template/src/forms/Prerequisites.ts

What are we suppose to follow if not the reference and other existing approved templates ?
This is also in the Figma reviewed and approved by Norda.

Reply:
All of these are easy fixes (fake steps, as, any etc.) and I'll remove this. Its not about whether I agree or disagree about them, I am indifferent to the fake steps (I agree the code looks hacky but I'd argue having the steps visible for the user isn't bad) and I think reducing "as" and definitely "any" is a positive, However my concern is with the communication of these standards.

Are these guidelines documented somewhere ? if they haven’t been why is this not communicated to us. Norda knows we are always working in the application system and its such waste of time to develop an application and not know until PR stage that there are guideline changes to then go and clean it up. Early awareness of these changes would allow us to incorporate them proactively, avoiding rework and this back and forth in PRs. Allowing the PRs to hopefully focus on bigger issues

I'd also mention that these same guidelines ("as" for example) is not being enforced consistently as seen in the TR Application PR posted at the same time, meaning we can't really trust the newest applications even. I would hope these guidelines would be a uniform decision within Norda and from (whenever guideline change occurs) x, y, z should now be enforced in all new applications, which would help build trust in the guidelines.